mysql-test/suite/innodb/r/log_encrypt_2_rk.result

```diff
@@ -66,6 +66,9 @@ c1	c2
 SELECT rotate_system_key("percona_redo");
 rotate_system_key("percona_redo")
 1
+SELECT rotate_system_key("percona_redo");
+rotate_system_key("percona_redo")
+1
 ALTER INSTANCE ROTATE INNODB MASTER KEY;
```

Comment:
is the duplication intentional?

Reply:
yes, I am checking if key was rotated correctly from version 2 to 3.